rpc/rpcserver/server.go

```diff
@@ -2034,12 +2051,14 @@ func (s *loaderServer) OpenWallet(ctx context.Context, req *pb.OpenWalletRequest
 		pubPassphrase = []byte(wallet.InsecurePubPassphrase)
 	}
 
-	_, err := s.loader.OpenExistingWallet(pubPassphrase)
+	wallet, err := s.loader.OpenExistingWallet(pubPassphrase)
```

Comment:
let's use w instead of wallet here to avoid an ambiguity with the wallet package.